src/main/java/com/casadocodigo/site/cadastrocategoria/NovaCategoriaRequest.java

```diff
@@ -0,0 +1,16 @@
+package com.casadocodigo.site.cadastrocategoria;
+
+import javax.validation.constraints.NotBlank;
+
+class NovaCategoriaRequest {
```

Comment:
Como esse projeto não é uma API, costumamos a usar como conveção o sufixo `Form`. Foi o que você usou no `NovoAutorForm`.

Faltou consistência no nome dessa classe.

```suggestion
class NovaCategoriaForm
```